app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
+require 'lighthouse/veterans_health/client'
+
+module FastTrack
+  class DisabilityCompensationJob
+    include Sidekiq::Worker
+
+    extend SentryLogging
+    # TODO: This is apparently at most about 4.5 hours.
+    # https://github.com/mperham/sidekiq/issues/2168#issuecomment-72079636
+    sidekiq_options retry: 8
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      submission_id = msg['args'].first
+      submission = Form526Submission.new
+      submission.start_evss_submission(_status, submission_id: submission_id)
+    end
+
+    def perform(form526_submission_id, full_name)
+      form526_submission = Form526Submission.find(form526_submission_id)
+      icn = Account.where(idme_uuid: form526_submission.user_uuid).first.icn
+
+      client = Lighthouse::VeteransHealth::Client.new(icn)
+      observations_response = client.get_resource('observations')
+      medicationrequest_response = client.get_resource('medications')
+
+      begin
+        bpreadings = FastTrack::HypertensionObservationData.new(observations_response).transform
+        return if no_recent_bp_readings(bpreadings)
+
+        medications = FastTrack::HypertensionMedicationRequestData.new(medicationrequest_response).transform
+
+        bpreadings = bpreadings.filter { |reading| reading[:issued].to_date > 1.year.ago }
+
+        bpreadings = bpreadings.sort_by { |reading| reading[:issued].to_date }.reverse!
+        medications = medications.sort_by { |med| med[:authoredOn].to_date }.reverse!
+
+        pdf = FastTrack::HypertensionPdfGenerator.new(full_name, bpreadings, medications, Time.zone.today).generate
+
+        FastTrack::HypertensionUploadManager.new(form526_submission).handle_attachment(pdf.render)
+
+        FastTrack::HypertensionSpecialIssueManager.new(form526_submission).add_special_issue
+      rescue => e
```

Comment:
🤔  ya ... I totally agree. I think I decided to capture everything to start because I legitimately don't know what errors we should be catching atm. Part of that is because we have such a small set of data in sandbox that we were using to test the parsing. So, I'm anticipating some errors coming from the classes that get called by this job bubbling up. 

I'll take a min to think about it but suggestions would be helpful. My goal had been to capture everything while we're testing and monitoring it then adding some more nuanced error handling after we know what it looks like out in the wild 🦓 .

Reply:
maybe NoMethodError?

Reply:
I would maybe use `StandardError` in this case but I do see where you are coming from with not knowing the specific exceptions.

Reply:
Actually now that I think about it, i think the way you have it no is shorthand for `StandardError` so it should be fine the way you have it now.

Reply:
😆  that's what I thought too but 🤷  I updated it. It's better to be explicit anyway.